File: examples/hello_world.js
// Example: Hello World in JavaScript
// This shows how to add JavaScript code to the sz project

/**
 * Returns a greeting message
 * @param {string} name - The name to greet (defaults to "World")
 * @returns {string} A greeting message
 */
function helloWorld(name = "World") {
    return `Hello, ${name}!`;
}

// Example usage
if (typeof module !== 'undefined' && module.exports) {
    // Node.js environment
    module.exports = helloWorld;
} else {
    // Browser environment
    console.log(helloWorld());
    console.log(helloWorld("sz Project"));
}

File: examples/hello_world.py
#!/usr/bin/env python3
"""
Example: Hello World
This is a simple example showing how to add code to the sz project.
"""

def hello_world(name="World"):
    """
    Returns a greeting message.
    
    Args:
        name (str): The name to greet. Defaults to "World".
    
    Returns:
        str: A greeting message.
    """
    return f"Hello, {name}!"

if __name__ == "__main__":
    # Example usage
    print(hello_world())
    print(hello_world("sz Project"))